app/boards/arm/waveshare_rp2040_zero/waveshare_rp2040_zero.dts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <rpi_pico/rp2040.dtsi>
+#include "waveshare_rp2040_zero-pinctrl.dtsi"
+#include <freq.h>
+
+/ {
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart0;
```

Comment:
For boards added to ZMK directly, we usually set the console to be the CDC ACM node added to the usb node. See https://github.com/zmkfirmware/zmk/blob/main/app/boards/arm/nice_nano/nice_nano.dtsi#L64 for an example node.